src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -68,11 +69,13 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
     const [elements, setElements] = useState(
         !storedElements ? [] : storedElements
     );
-    const defaultPosition = 100;
+    let defaultPositionX = 100;
+    let defaultPositionY = 100;
```

Comment:
These will get reset to the default values each time the component re-renders.

Maybe we return these from the positionLookUp method instead?

Reply:
I was counting on that to happen, so, the LookUp method will fill the gaps.